src/actions/vstudio/vs2005_solution.lua

```diff
@@ -139,6 +146,62 @@
 
 
 --
+-- Write out the list of project configuration platforms.
+--
+
+	sln2005.elements.projectConfigurationPlatforms = function(cfg, context)
+		return {
+			sln2005.activeCfg,
+			sln2005.build0,
+		}
+	end
+
+
+	function sln2005.projectConfigurationPlatforms(wks, sorted, descriptors)
+		p.w("GlobalSection(ProjectConfigurationPlatforms) = postSolution")
+		local tr = p.workspace.grouptree(wks)
+		tree.traverse(tr, {
+			onleaf = function(n)
+				local prj = n.project
+				table.foreachi(sorted, function(cfg)
+					local context = {}
+					-- Look up the matching project configuration. If none exist, this
+					-- configuration has been excluded from the project, and should map
+					-- to closest available project configuration instead.
+					context.prj = prj
```

Comment:
While there's no harm in doing it this way, you can also access the project in the handler functions via `cfg.project`.